src/libexpr/primops.cc

```diff
@@ -547,32 +547,71 @@ typedef list<Value *> ValueList;
 #endif
 
 
+static Bindings::iterator extractAttrNameFromPrimopCall(
```

Comment:
Not sure I understand the name of this function. It doesn't seem to have anything to do with primops specfically, and it extracts an attr, not an attr name (which is given). Maybe just `getAttr`?